app/assets/javascripts/admin/admin_ofn.js.coffee

```diff
@@ -1,3 +1,3 @@
-angular.module("ofn.admin", ["ngResource", "ngAnimate", "admin.utils", "admin.indexUtils", "admin.dropdown", "admin.products", "admin.taxons", "infinite-scroll"]).config ($httpProvider) ->
+angular.module("ofn.admin", ["admin.enterpriseRoles", "ngResource", "ngAnimate", "admin.utils", "admin.indexUtils", "admin.dropdown", "admin.products", "admin.taxons", "infinite-scroll"]).config ($httpProvider) ->
```

Comment:
I'd rather keep this separate to the `ofn.admin` module, and instead change `app/views/admin/enterprise_roles/_data.html.haml` so that all data there is provided to the `admin.enterpriseRoles` module. This would also require using `admin.enterpriseRoles` as the `ng-app` in `/Users/rob/projects/openfoodnetwork/app/views/admin/enterprise_roles/index.html.haml` and moving `AdminEnterpriseRolesCtrl` to the `admin.enterpriseRoles` module.

Perhaps we can do all of that in a separate PR, don't worry about doing it now and confusing things here.